src/react/src/components/InitialClaimFlow/ClaimForm/hooks.js

```diff
@@ -128,8 +129,13 @@ export const useClaimForm = (
             field => formik.errors[field],
         );
 
-        // Only disable button if user has interacted AND there are errors.
-        return hasInteractedWithCurrentStep && hasCurrentStepErrors;
+        // Disable button if:
+        // 1. User has interacted with fields AND there are Formik validation errors, OR
+        // 2. There are emissions validation errors
+        return (
```

Comment:
This has been updated in https://github.com/opensupplyhub/open-supply-hub/pull/780, pls sync the change.